tgui/packages/tgui/interfaces/ChemRequestReceiver.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @file
+ * @copyright 2022
+ * @author LeahTheTech (https://github.com/TobleroneSwordfish)
+ * @license MIT
+ */
+
+import { useBackend, useLocalState, useSharedState } from '../backend';
+import { Button, LabeledList, Input, Section, Flex, Box, Stack, Tabs } from '../components';
+import { Window } from '../layouts';
+
+import { capitalize } from '../../common/string';
+
+const ChemRequest = (props, context) => {
+  const { act } = useBackend(context);
+  const {
+    name,
+    id,
+    reagent_name,
+    reagent_color,
+    volume,
+    notes,
+    area,
+    state,
+    interactable,
+    age,
+  } = props;
+  // some mild colour hackery to make the text visible on (hopefully) any reagent colour
+  const color_string = "rgba(" + reagent_color[0] + "," + reagent_color[1] + ", " + reagent_color[2] + ", 1)";
+  const lightness = reagent_color.reduce((a, b) => a + b, 0)/3;
+  return (
+    <Section height>
+      <Flex direction="column" height={9}>
+        <Flex.Item grow={1}>
+          <Stack vertical>
+            <Stack.Item>{name} requested</Stack.Item>
+            <Stack.Item align="center"><Box width={16} textAlign="center" backgroundColor={color_string} color={lightness > 255/2 ? "black" : "white"}>{capitalize(reagent_name)} ({volume}u)</Box></Stack.Item>
+            <Stack.Item>from {area} {age} ago. <br /> {notes && `Notes: ${notes}`}</Stack.Item>
+          </Stack>
+        </Flex.Item>
+        <Flex.Item>
+          <Box>
+            {state === "pending" && (
+              <>
+                <Button disabled={!interactable} align="center" width="49.5%" color="red" icon="ban" onClick={() => { act("deny", { id: id }); }}>Deny</Button>
```

Comment:
Use `Stack` or `Flex` to put these buttons horizontally next to each other

Reply:
But they're fixed width, not sure how a stack would help?